README.md

```diff
@@ -52,7 +52,7 @@ cb
 #> [1] "a,b\r\n1,4\r\n2,5\r\n3,6" # on Windows
 ```
 
-`xl2df_clip` allows clips from spreadsheets containing matrices to be read in as data.frames directly. It can be exactly like `read_clip`, directly after the copy step, and also after the output of read_clip has been assigned to a variable.
+`read_clip_tbl` allows clips from spreadsheets containing matrices to be read in as data.frames directly. It can be exactly like `read_clip`, directly after the copy step, and also after the output of read_clip has been assigned to a variable.
```

Comment:
I would keep only this line in README.md, and move the longer explanation into the function documentation.